files/build_templates/init_cfg.json.j2

```diff
@@ -38,7 +38,7 @@
 {%- set features = [("bgp", "{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}disabled{% else %}enabled{% endif %}", false, "enabled"),
                    ("database", "always_enabled", false, "always_enabled"),
                    ("lldp", "enabled", true, "enabled"),
-                   ("pmon", "enabled", true, "enabled"),
+                   ("pmon", "enabled", "{% if 'type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'SpineRouter' %}false{% else %}true{% endif %}", "enabled"),
```

Comment:
This PR works with PR https://github.com/sonic-net/sonic-host-services/pull/135
But delayed value "true/false" are the match the expected value which in the featured line 79.  The defined value should be "True/False".
This change causes the system fails to start PMON

Reply:
Good catch, I suppose it's because of the removal of 'lower()' function, not sure why Abhishek changed this behavior.
Let me draft another PR and see.

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/19657